base/reflection.jl

```diff
@@ -547,7 +547,7 @@ end
     isprimitivetype(T) -> Bool
 
 Determine whether type `T` was declared as a primitive type
-(i.e. using the `primitive` keyword).
+(i.e. using the `primitive type` keyword).
```

Comment:
```suggestion
(i.e. using the `primitive type` syntax).
```
maybe? Technically `primtive` and `type` are keywords, but together they're a syntax form. :bike: